CHANGELOG.md

```diff
@@ -56,6 +57,131 @@
+patches](https://github.com/Kong/kong-build-tools/tree/master/openresty-build-tools/patches)
+(and, as highlighted above, compile OpenResty with the new
+lua-kong-nginx-module). Our [kong-build-tools](https://github.com/Kong/kong-build-tools)
+repository will allow you to do both easily.
+
+- Bump luarocks from 3.4.0 to 3.5.0.
+  [#6699](https://github.com/Kong/kong/pull/6699)
+- Bump luasec from 0.9 to 1.0.
+  [#6814](https://github.com/Kong/kong/pull/6814)
+- Bump lua-resty-dns-client from 5.2.1 to 5.2.3.
+  [#6937](https://github.com/Kong/kong/pull/6937)
+- Bump kong-lapis from 1.8.1.2 to 1.8.3.1.
+  [#6925](https://github.com/Kong/kong/pull/6925)
+- Bump pgmoon from 1.11.0 to 1.12.0.
+  [#6741](https://github.com/Kong/kong/pull/6741)
+- Bump lua-resty-openssl from 0.6.11 to 0.7.0.
+  [#6862](https://github.com/Kong/kong/pull/6862)
+- Bump kong-plugin-zipkin from 1.2 to 1.3.
+  [#6936](https://github.com/Kong/kong/pull/6936)
+- Bump kong-prometheus-plugin from 1.0 to 1.2.
+  [#6958](https://github.com/Kong/kong/pull/6958)
+
+### Additions
+
+##### Core
+
+- Relaxed version check between Control Planes and Data Planes, allowing
+  Data Planes that are missing minor updates to still connect to the
+  Control Plane. Also, now Data Plane is allowed to have a superset of Control
+  Plane plugins. [6932](https://github.com/Kong/kong/pull/6932)
+
+##### PDK
+
+- Beta release of Protobuf plugin communication protocol, which can be used in
+  place of MessagePack to communicate with non-Lua plugins.
+  [6941](https://github.com/Kong/kong/pull/6941)
+
+##### Plugins
+
+- zipkin: support for Jaeger style uber-trace-id headers.
+  [101](https://github.com/Kong/kong-plugin-zipkin/pull/101)
+  Thanks [nvx](https://github.com/nvx) for the patch!
+- zipkin: support for OT headers.
+  [103](https://github.com/Kong/kong-plugin-zipkin/pull/103)
+  Thanks [ishg](https://github.com/ishg) for the patch!
+- zipkin: allow insertion of custom tags on the Zipkin request trace.
+  [102](https://github.com/Kong/kong-plugin-zipkin/pull/102)
+- zipkin: creation of baggage items on child spans is now possible.
+  [98](https://github.com/Kong/kong-plugin-zipkin/pull/98)
+  Thanks [Asafb26](https://github.com/Asafb26) for the patch!
+- Several plugins: capability to set new log fields, or unset existing fields,
+  by executing custom Lua code in the Log phase.
+  [6944](https://github.com/Kong/kong/pull/6944)
+
+### Fixes
+
+##### Core
+
+- Changed default values and validation rules for plugins that were not
+  well-adjusted for dbless or hybrid modes.
+  [6885](https://github.com/Kong/kong/pull/6885)
+- Topological sort now prioritizes core, avoiding problems when plugin entities
+  use core entities but don't explicitly depend on them.
+  [6880](https://github.com/Kong/kong/pull/6880)
+- If needed, `Host` header is now updated between balancer retries, using the
+  value configured in the correct upstream entity.
+  [6796](https://github.com/Kong/kong/pull/6796)
```

Comment:
```suggestion
- If needed, `Host` header is now updated between balancer retries.
  [6796](https://github.com/Kong/kong/pull/6796)
```

I would change this changelog entry simply to:

"If needed, `Host` header is now updated between balancer retries."

So I'd remove the "using the value configured in the correct upstream entity." because I don't think it uses the `host_header` attribute from upstream entities, if that's what this part means.

I suggest this because there are inter relationships between host headers that are still in discussion regarding what's the precedence and how they work together. See https://github.com/Kong/kong/issues/5580

Please let me know what you think